spec/jobs/mail_forward_job_spec.rb

```diff
@@ -18,7 +18,7 @@
 
     it do
       expect(fake_http).to have_received(:post).with(
-        message_url, form: { to: mail_alias.user.email }
+        message_url, form: { to: 'testing@alpha.csvalpha.nl' }
```

Comment:
Isn't the old way here still possible? Like reading the email directly from mail_alias

Reply:
No, because the mail_alias in this case is `testing@csvalpha.nl` and we do forward to the `alpha` subdomain